src/modules/addRemove.js

```diff
@@ -0,0 +1,46 @@
+let descriptionBackup;
+
+export const taskInput = document.querySelector('.input-task');
+
+export const addTaskToArray = (task, tasks) => {
+  tasks.push(task);
+};
```

Comment:
* I think the property index can be set to the value of the array length  (i.e. if you're adding a 5th task to the list, the index of that task should equal 5) [Please refer to the project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).) 
* HINT 💭 You can set your index by doing (index: x.length + 1 ) so that items created don't take the index `0`

Reply:
If you check the caller function in the index file you'll notice the elements in the list take index from the array:

```suggestion
const setTaskInput = () => {
  taskInput.addEventListener('keyup', (event) => {
    if (event.key === 'Enter') {
      const newInput = {
        description: event.target.value,
        completed: false,
        index: tasksArray.length,
      };
      event.target.value = '';
      addTaskToArray(newInput, tasksArray);
      displayList();
      storeListData();
    }
  });
};
```

Reply:
Now the coded is changed to be:

```suggestion
const setTaskInput = () => {
  taskInput.addEventListener('keyup', (event) => {
    if (event.key === 'Enter') {
      const newInput = {
        description: event.target.value,
        completed: false,
        index: tasksArray.length + 1,
      };
      event.target.value = '';
      addTaskToArray(newInput, tasksArray);
      displayList();
      storeListData();
    }
  });
};

const updateArray = () => {
  const items = [...tasksList.children];

  tasksArray = [];

  items.forEach((item, index) => {
    const childDiv = item.firstChild;
    const newTask = {
      description: childDiv.lastChild.textContent,
      completed: childDiv.firstChild.checked,
      index: index + 1,
    };
    tasksArray.push(newTask);
  });
  storeListData();
};
```

Reply:
hi @alvp01, the list in the array (i.e, if it is the 5th item, should have an index of 5), hope you understand if not reply this comment 👍

Reply:
* done 👍